gtnet/utils.py

```diff
@@ -1,24 +1,49 @@
-import os
 from pkg_resources import resource_filename
+from .sequence import _get_DNA_map
+import pandas as pd
+import ruamel.yaml as yaml
 import json
 import logging
 import sys
+import os
 
 
-def get_species_pred(output):
-    return output.mean(axis=0).argmax()
+class gtnet_config:
```

Comment:
```suggestion
class GTNetConfig:
```

Reply:
Can you add a docstring to this class?

Reply:
Also, user classes are typically named with camel case. It's just a Python convention that makes maintenance easier.